live/engine/order_registry.py

```
@@ -48,7 +48,40 @@
+    env_id = os.environ.get("SH_RUN_ID")
+    if env_id:
+        return env_id
+
+    lock_file = LIVE_ROOT / "run" / ".current_run_id"
+    try:
+        if lock_file.exists():
+            mtime = lock_file.stat().st_mtime
+            if time.time() - mtime < 43200:  # 12h
+                return lock_file.read_text().strip()
+    except Exception:
+        pass
```

Ruff (0.16.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)